babel.config.js

```diff
@@ -1,25 +1,25 @@
 module.exports = {
-  "sourceMaps": true,
-  "presets": [
-    ["@babel/preset-env", {
-      "targets": {
-        "browsers": [
-          "last 2 versions"
-        ]
-      }
-    }],
-    "@babel/preset-typescript",
-    "@babel/preset-flow",
-    "@babel/preset-react"
-  ],
-  "plugins": [
-    ["@babel/plugin-proposal-decorators", {"legacy": true}],
-    "@babel/plugin-proposal-export-default-from",
-    "@babel/plugin-proposal-numeric-separator",
-    "@babel/plugin-proposal-class-properties",
-    "@babel/plugin-proposal-object-rest-spread",
-    "@babel/plugin-transform-flow-comments",
-    "@babel/plugin-syntax-dynamic-import",
-    "@babel/plugin-transform-runtime"
-  ]
+    "sourceMaps": true,
+    "presets": [
+        ["@babel/preset-env", {
+            "targets": {
+                "browsers": [
```

Comment:
Fwiw they're removing this soon in favour of just setting 'targets' (https://babeljs.io/docs/en/babel-preset-env#targetsbrowsers) - we may as well do it now.

Reply:
so our `modules` is at the wrong level?